examples/flutter_timer/lib/bloc/timer_state.dart

```diff
@@ -9,27 +9,27 @@ abstract class TimerState extends Equatable {
   List<Object> get props => [duration];
 }
 
-class Ready extends TimerState {
-  const Ready(int duration) : super(duration);
+class TimerInitial extends TimerState {
+  const TimerInitial(int duration) : super(duration);
 
   @override
   String toString() => 'Ready { duration: $duration }';
 }
 
-class Paused extends TimerState {
-  const Paused(int duration) : super(duration);
+class TimerRunningPaused extends TimerState {
```

Comment:
```suggestion
class TimerRunPause extends TimerState {
```

Reply:
Well, Run can be used as verb didn't cross my mind. Great suggestion here @felangel . Although, for this particular state, I believe, it should be paused state rather than pause state.

Reply:
Thanks! The reason I suggested TimerRunPause is because TimerRunPaused is past tense and reads like an event.

Reply:
Makes sense since we also have TimerPaused. So, I will go ahead and push a new commit resolving all these changes.